src/modules/home/contexts/accommodation.context.tsx

```diff
@@ -0,0 +1,50 @@
+import { createContext, type ReactNode, useContext, useState } from 'react';
+import { type AccommodationFormData } from '../types/accommodation-form-data.interface';
+import { type AccommodationBasics } from '../types/accommodation-basics.type';
+
+export interface AccommodationContextType {
+  data: AccommodationFormData;
+  updateData: (newData: Partial<AccommodationFormData>) => void;
+  basics: AccommodationBasics;
+  updateBasics: (newBasics: Partial<AccommodationBasics>) => void;
+}
+
+const AccommodationContext = createContext<AccommodationContextType>({
```

Comment:
you can also just make its type <AccommodationContextType | null> and set it to null initially instead of this